CONTRIBUTING.md

```diff
@@ -51,3 +51,35 @@ JSON is formatted using Prettier.
 **Use the `.json` file extension where possible**: configuration files such as `.eslintrc.json` have an explicit file extension and is not called simply `.eslintrc` to ensure all code formatting rules can be applied automatically.
 
 **Avoid code formatting in generated JSON files**: files generated by tools, such as `package.json` that regularly updated using `npm install`, should be excempt from automatic formatting. Configure these files in `.prettierignore`.
+
+## Contributing as a designer or content specialist
+
+We encourage contributions from designers, copywriters and other stakeholders. There are many different ways to make a useful contribution, from the addition of (UX) documentation, new design components to fixing typing errors.
+
+In order to contribute you must have a [GitHub account](https://github.com/join).
+
+### Using GitHub
+
+GitHub allows the NL Design System to be maintained in the open. Hosting the content on GitHub has several benefits; all the content is freely available to everything in a central place and can accept contributions from everyone. Version control means that a full history is kept of the progress of and changes to the design system. Changes can be reverted when needed.
+
+The easiest way to work with GitHub is through the [GitHub Desktop application](https://desktop.github.com/), available for both Mac OS-X and Windows.
+
+For both new additions and changes to existing components or documentation you first need to create a ‘branch’. A branch duplicates the ‘repository’ (all content of NLDS), and allows you to make changes or additions safely in isolation without impacting the rest of the project.
+
+Once your work is finished you are able to ‘commit’ your changes first to your branch. It is important to describe as clear and concise as possible what you have changed or added in the commit message. This message will be displayed to reviewers and all other contributors of the design system.
+
+NLDS uses [Conventional Commits](https://www.conventionalcommits.org/) in order for commit messages to follow a consistent pattern.
```

Comment:
I suggest we always write "NL Design System" in full in the documentation instead of "NLDS", or refer to "the project" if you want to avoid repetition. The fewer acronyms the better!

Reply:
Perhaps explicitly state that commit messages must **always** start with the type, for example "`docs: `" before we can accept the commit. Perhaps if there is documentation on how to amend commit messages in the GitHub desktop app we can include a link to that.

Reply:
I have updated instances of NLDS. It is now either called ‘NL Design System’ or ‘the project’ where it makes the best sense in context.

Reply:
The commit section has been extended to elaborate on the type that must be stated in the commit message.